cmd/tools/grafana/grafana.go

```diff
@@ -400,10 +398,10 @@ func initImportVars() {
 
 	// default behaviour
 	if opts.dir == "grafana/dashboards" && opts.serverfolder.name == "" {
-		m[filepath.Join(opts.dir, "cmode")] = &Folder{name: "Harvest-" + harvestRelease + "-cDOT"}
-		m[filepath.Join(opts.dir, "cmode", "details")] = &Folder{name: "Harvest-" + harvestRelease + "-cDOT Details"}
-		m[filepath.Join(opts.dir, "7mode")] = &Folder{name: "Harvest-" + harvestRelease + "-7mode"}
-		m[filepath.Join(opts.dir, "storagegrid")] = &Folder{name: "Harvest-" + harvestRelease + "-StorageGrid"}
+		m[filepath.Join(opts.dir, "cmode")] = &Folder{name: "Harvest-main-cDOT"}
```

Comment:
If we're ditching versioned folders maybe `Harvest-cDOT`, `Harvest-StorageGrid`, etc. would be better?

Reply:
yeah, I was just matching the behaviour with what our docker workflow currently doing.